modules/gdscript/gdscript.cpp

```diff
@@ -1966,6 +1966,17 @@ void GDScriptLanguage::add_named_global_constant(const StringName &p_name, const
 	named_globals[p_name] = p_value;
 }
 
+Variant GDScriptLanguage::get_any_global_constant(const StringName &p_name) {
+	ERR_FAIL_COND_V(!named_globals.has(p_name) && !globals.has(p_name), Variant());
```

Comment:
Isn't `has` called two times here? First in the check, than below again?

Reply:
Yes, it is - the first time to raise an error, the second time to disambiguate which map it should find the map from.

I did want an error to be raised here, but it is slightly more inefficient this way. That being said, right now this is only used in the gdscript analyzer, which runs once for every script and that's it, so I figured it was OK?

Reply:
I don't think it is a major problem here, since `has` is very fast in `HashMap`s.
But can't you just raise an error in line 1977, after you tried looking up the value in both `HashMap`?

Reply:
Oh yeah, good point :) I'll do that!

Reply:
Done! And thanks for checking this out! :)